packages/cli/test/unit/commands/env/add.test.ts

```diff
@@ -64,7 +64,7 @@ describe('env add', () => {
           },
           {
             key: `argument:environment`,
-            value: '[REDACTED]',
+            value: 'preview',
```

Comment:
Is there a test case for when `--environment` is a redacted value?

Reply:
Good catch, added a test case!